src/newton.jl

```diff
@@ -97,7 +97,7 @@ function newton_{T}(df::AbstractDifferentiableMultivariateFunction,
     end
     function fgo!(xlin::Vector, storage::Vector)
         go!(xlin, storage)
-        return(dot(fvec, fvec) / 2)
+        dot(fvec, fvec) / 2
```

Comment:
This is just a personal style quirk. I don't like making return look like a function.

So I was going to change this to `return dot(fvec, fvec) / 2`. But then because I was making a change anyway I figured I might as well go all the way and adopt my preferred style of implicit return